libs/framework/gtest/src/CxxBundleContextTestSuite.cc

```diff
@@ -78,6 +78,7 @@ TEST_F(CxxBundleContextTestSuite, RegisterServiceTest) {
     }
 
     ctx->waitForEvents();
+    ctx->waitIfAbleForEvents();
```

Comment:
Did you forget to remove line 80?

Reply:
yes, and will remote the superfluous wait entries.